### PR TITLE
fix: feature db name to match release name

### PR DESCRIPTION
**Pull request by rafasdc**
Implements [NDT-]

- [ ] Check to trigger automatic release process

### CODE REVIEW

File: .github/workflows/deploy_feature.yaml [hunk 1]
```diff
@@ -62,6 +62,8 @@ jobs:
           openshift_token: ${{ secrets.OPENSHIFT_TOKEN }}
       - name: Setup database
         run: |
+          FEATURE_NAME_LOWER=$(echo $FEATURE_NAME | tr '[:upper:]' '[:lower:]')
+          FEATURE_NAME_LOWER_SHORT=$(echo $FEATURE_NAME_LOWER | cut -c -30)
```
Comment (AntBush) [inline .github/workflows/deploy_feature.yaml:66]: Aren't we missing the `sed 's/-*$//'` command here? 
```suggestion
          FEATURE_NAME_LOWER_SHORT=$(echo $FEATURE_NAME_LOWER | cut -c -30 | sed 's/-*$//')
```

File: .github/workflows/deploy_feature.yaml [hunk 2]
```diff
@@ -62,6 +62,8 @@ jobs:
...
           chmod +x ./lib/feature_envs/create_feature_db.sh
           ./lib/feature_envs/create_feature_db.sh ccbc ${{ env.FEATURE_NAME }} ${{ secrets.OPENSHIFT_APP_NAMESPACE }}
```
Comment (AntBush) [inline .github/workflows/deploy_feature.yaml:68]: I think we're supposed to pass the `FEATURE_NAME_LOWER_SHORT` here right? 

```suggestion
          ./lib/feature_envs/create_feature_db.sh ccbc "$FEATURE_NAME_LOWER_SHORT" ${{ secrets.OPENSHIFT_APP_NAMESPACE }}
```

Comment (rafasdc) [reply .github/workflows/deploy_feature.yaml:68]: Yup, fixed!

File: .github/workflows/deploy_feature.yaml (hunk 1 above)
Comment (rafasdc) [reply .github/workflows/deploy_feature.yaml:66]: Yes!

**Review by AntBush — APPROVED** [review]

**Pull request merged**